pkg/tui/table.go

```diff
@@ -115,6 +119,13 @@ func WithViewModeFunc(fn ViewModeFunc) TableOption {
 	}
 }
 
+// WithRefreshFunc sets a func that is triggered when a user press 'r'.
```

Comment:
Another one that requires update

Reply:
thanks for pointing it out for me @adolsalamanca